scss/_main.scss

```diff
@@ -1,41 +1,43 @@
-$redColor: #F1413D;
+$redColor   : #F1413D;
 $yellowColor: #FBB243;
-$greenColor: #328849;
+$greenColor : #328849;
 
 main {
-    width: 100%;
+    width : 100%;
     height: 88vh;
 
     #visualizer-container {
-        width: 90%;
-        margin: auto;
-        background: whitesmoke;
-        height: 88vh;
-        display: flex;
+        width          : 90%;
+        margin         : auto;
+        background     : whitesmoke;
+        height         : 88vh;
+        display        : flex;
+        justify-content: center;
+        gap            : .2rem;
 
         div {
-            border: 1px solid transparent;
+            border     : 1px solid transparent;
             margin-left: 1px;
-            background: rgb(225, 186, 253);
+            background : rgb(225, 186, 253);
```

Comment:
I don't understand what is the need for aligning the colons vertically. It is causing some extra spaces. Can you please revert these colons spacing changes?

Reply:
Oh, it's due to formater I will fix it.